### PR TITLE
Add shops from osm.org and iD

### PR DESCRIPTION
**Pull request by pnorman**
- [x] Update tests
- [x] Update data migrations
- [x] Update docs

This is a big part of #1425

### CODE REVIEW

File: yaml/pois.yaml [hunk 1]
```diff
@@ -1360,8 +1360,11 @@ filters:
       <<: *output_properties
       kind: cycle_barrier
   - filter:
-      shop: [bakery, books, butcher, car, car_repair, clothes, computer, convenience,
-        fashion, florist, gift, greengrocer, hairdresser, jewelry, mobile_phone, optician, pet]
+      shop: [art, bakery, beauty, books, butcher, car, car_repair, clothes,
```
Comment (nvkelso) [inline yaml/pois.yaml]: `shop=coffee` and `shop=deli` seem more like `amenity=cafe` and should get that **min_zoom** range.

`shop=variety_store` and `shop=furniture` and `shop=shoes` seems more like `craft: [brewery, carpenter...]` and should get that **min_zoom** range.

I'm fine with the rest being `min_zoom: 17`.

**Comment by pnorman** [comment]
Pushed in a missing file, haven't handled the review comments yet

File: yaml/pois.yaml (hunk 1 above)
Comment (pnorman) [reply yaml/pois.yaml]: > shop=variety_store and shop=furniture and shop=shoes seems more like craft: [brewery, carpenter...] and should get that min_zoom range. Even though this has been around awhile, shop=pet should also be in this set.

I'm making the changes, but `craft` is a bit of a weird thing in OSM, and a better word would probably have been `trade`. I don't see the `craft` values and those `shop` values forming much of a coherent set - some are places a customer goes to buy stuff, others don't interact directly with customers at their locations.

**Comment by pnorman** [comment]
Everything should be good, and PEP style issues are fixed.

**Pull request merged**